dma4500mSup/dma4500m.db

```diff
@@ -0,0 +1,160 @@
+record(bo, "$(P):START") {
+  field(DESC, "Start measuring")
+  field(DTYP, "stream")
+  field(OUT, "@dma4500m.proto start $(PORT)")
+  field(DISV, "1")
+  field(SDIS, "$(P):_MEASURING")
+  field(FLNK, "$(P):_START_MEASURING")
+}
+
+alias("$(P):START", "$(P):START:SP")
+
+## set to > 0 for auto measurement
+record(longin, "$(P):AUTOMEASURE:FREQ")
+{
+  field(DESC, "Automeasurement frequency; 0 to disable")
+  field(VAL, 0)
+  field(EGU, "s")
+  info(autosaveFields, "VAL")
+}
+
+alias("$(P):AUTOMEASURE:FREQ", "$(P):AUTOMEASURE:FREQ:SP")
+
+record(calcout, "$(P):AUTOMEASURE")
+{
+  field(INPA, "$(P):AUTOMEASURE:FREQ NPP")
+  field(CALC, "A=0?1:VAL=-1?A:VAL-1")
+  field(OOPT, "When Zero")
+  field(DOPT, "1")
+  field(OUT, "$(P):START PP")
+  field(SDIS, "$(P):_MEASURING")
+  field(DISV, "1")
+  field(SCAN, "1 second")
+}
+
+record(fanout, "$(P):_START_MEASURING") {
+  field(LNK1, "$(P):_UPDATE_FLAGS")
+  field(LNK2, "$(P):MEASUREMENT")
+  info(archive, "VAL")
+}
+
+record(fanout, "$(P):_UPDATE_FLAGS") {
+  field(LNK1, "$(P):_MEASURING")
+  field(LNK2, "$(P):_MONITOR_STATE_CHANGE")
+}
+
+record(calc, "$(P):_MEASURING") {
+  field(INPA, "$(P):START")
+  field(INPB, "$(P):MEASUREMENT")
+  field(CALC, "A == 1 || B == 1")
+  field(FLNK, "$(P):_RESET_START")
+  info(archive, "VAL")
+}
+
+record(bo, "$(P):_RESET_START") {
+  field(VAL, "0")
+  field(OUT, "$(P):START.VAL")
+  info(archive, "VAL")
+}
+
+record(calcout, "$(P):_MONITOR_STATE_CHANGE") {
+  field(INPA, "$(P):MEASUREMENT")
+  field(CALC, "A")
+  field(OOPT, "On Change")
+  field(DOPT, "Use OCAL")
+  field(OCAL, "A=2?1:0")
+  field(OUT, "$(P):_DATA_AVAILABLE PP")
+  field(FLNK, "$(P):_GET_DATA")
+}
+
+record(bi, "$(P):_DATA_AVAILABLE") {
+  field(VAL, "0")
+}
+
+record(mbbi, "$(P):MEASUREMENT") {
```

Comment:
This PV produced an error in my IOC log file, it looks like there is perhaps a mismatch between the emulator's return statement and the protocol inputs?

Reply:
I couldn't reproduce this with the current IOC; are you running it against the real device or the emulator? I remember having some "mismatched input" errors when running against the real device before I increased the reply timeout